libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/MapPaddingAdjustor.java

```diff
@@ -13,47 +13,68 @@
   private static final int BOTTOMSHEET_PADDING_MULTIPLIER = 4;
   private static final int WAYNAME_PADDING_MULTIPLIER = 2;
 
-  private final int defaultTopPadding;
-  private final int waynameTopPadding;
-  private MapboxMap mapboxMap;
+  private final MapboxMap mapboxMap;
+  private final int[] defaultPadding;
+  private int[] customPadding;
 
   MapPaddingAdjustor(MapView mapView, MapboxMap mapboxMap) {
     this.mapboxMap = mapboxMap;
-    defaultTopPadding = calculateTopPaddingDefault(mapView);
-    waynameTopPadding = calculateTopPaddingWithWayname(mapView.getContext(), defaultTopPadding);
+    defaultPadding = calculateDefaultPadding(mapView);
   }
 
-  void updateTopPaddingWithWayname() {
-    updateTopPadding(waynameTopPadding);
+  // Testing only
+  MapPaddingAdjustor(MapboxMap mapboxMap, int[] defaultPadding) {
+    this.mapboxMap = mapboxMap;
+    this.defaultPadding = defaultPadding;
+  }
+
+  void updatePaddingWithDefault() {
+    customPadding = null;
+    updatePaddingWith(defaultPadding);
+  }
+
+  void updatePaddingWithZero() {
```

Comment:
Minor comment - What about "exposing" (package private) `updatePaddingWith` instead and removing this method?